cumulusci/tasks/salesforce/__init__.py

```diff
@@ -54,3 +55,33 @@
 from cumulusci.tasks.salesforce.UninstallPackagedIncremental import (
     UninstallPackagedIncremental,
 )
+
```

Comment:
@davisagli  Do we have a preference for this style of fix or just overriding flake8 with --no-verify when it balks at `__init__.py`?

Reply:
I told Matthew to do it this way, since using `--no-verify` doesn't seem very sustainable. I'd rather configure pyflakes to skip the F401 check just for this file, but I don't think it gives us a way to do that.